### DISCUSSION TITLE
Did I get approved or rejected? GH Students website says yes but mail says no

### DISCUSSION THREAD

**Discussion by Stef-00012** [community/community#147865] [GitHub Education] [answered]
### Hi everyone,

I have applied for the GitHub students pack and according to https://education.github.com/discount_requests/application website I got approved
![image](https://github.com/user-attachments/assets/5cfa9774-02f5-451c-a80a-e21545bf4936)
![image](https://github.com/user-attachments/assets/c81c9530-b283-4913-bf9f-2a6f6e16ab48)

But if I open the mail I received it says I got rejected, which is correct now?
![image](https://github.com/user-attachments/assets/e12cdfd4-95c9-42b6-87ab-fdcb4478d869)

Did I get approved or rejected?

**Comment by davevad93** [comment, marked as answer]
Hi @Stef-00012 ,

The first screenshot is correct, you have been approved. 
When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Stef-00012 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
